rpm/SPECS/contextBroker.spec

```diff
@@ -178,7 +178,7 @@ fi
 - Add: transient entities functionality (new NGSIv2 builtin attribute: dateExpires) (#3000)
 - Add: "attrs" field in POST /v2/op/query (making "attributes" obsolete) (#2604)
 - Add: "expression" field in POST /v2/op/query (#2706)
-- Fix: large integer wrong rendering in responses (#2603)
+- Fix: large integer wrong rendering in responses (#2603, #2425, #2506)
```

Comment:
Typically we use CNR for this... but the fix was already included in a past version, so I'm modifying the full changelog in the .sepc file.

The release 1.15.0 has been also edited to add these issues.